tools/symbol_addrs.txt

```diff
@@ -327,6 +327,7 @@ func_800AAF10 = 0x800AAF10;
 gLevelID = 0x800AC0C0;
 gLevelNames = 0x800AC0C4;
 gPhotoCount = 0x800AC0E0;
+get_player_name = 0x800BF498;
```

Comment:
Sorry, this is my bad.. It should probably be getPlayerName.. Or in general, we should all discuss a naming convention scheme